admin-notice/class-admin-notice-controller.php

```diff
@@ -31,7 +31,7 @@ public function display_notices() {
 	}
 
 	public function enqueue_scripts() {
-		wp_enqueue_script( 'vip-admin-notice-script', plugins_url( '/js/script.js', __FILE__ ), [], '1.0', true );
+		wp_enqueue_script( 'vip-admin-notice-script', plugins_url( '/js/script.js', __FILE__ ), [ 'common' ], '1.1', true );
```

Comment:
Dismiss buttons get added by `common.js`; therefore, it makes sense to list it as a dependency. This also ensures that `common.js` will be loaded before `script.js`.